### PR TITLE
Add the schedules view with create, show and delete features

### PR DESCRIPTION
**Pull request by Naviunni**
Create the landing page with details of schedules
Schedules table has metadata - name and semester
* User can see schedule details by clicking on the card
* User can delete schedules
* User can create a schedule using name and semester
* User can search for schedules

*  Coverage: 98.96%

Tracker: https://www.pivotaltracker.com/story/show/188364866

### CODE REVIEW

File: app/assets/stylesheets/schedules.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* app/assets/stylesheets/schedules.css 
+This contains the css required for all the schedules views*/
+.container {
```
Comment (wahibkapdi) [inline app/assets/stylesheets/schedules.css]: Calling a class container may cause conflicts with the bootstrap container class.

File: bin/bundle [hunk 2]
```diff
@@ -18,7 +18,7 @@ m = Module.new do
   end
 
   def env_var_version
-    ENV['BUNDLER_VERSION']
+    ENV.fetch('BUNDLER_VERSION', nil)
```
Comment (wahibkapdi) [inline bin/bundle:21]: Is this caused due to version difference?

**Review by wahibkapdi — CHANGES_REQUESTED** [review]

File: bin/bundle (hunk 2 above)
Comment (colbyendres) [reply bin/bundle:21]: `fetch` allows for a default value if `BUNDLER_VERSION` isn't defined, which in this case is `nil`. This has the same behavior as `ENV['BUNDLER_VERSION']`.

Comment (Naviunni) [reply bin/bundle:21]: Rubocop made this change automatically

File: app/assets/stylesheets/schedules.css (hunk 1 above)
Comment (Naviunni) [reply app/assets/stylesheets/schedules.css]: Good point. I have removed this

**Review by wahibkapdi — APPROVED** [review]

**Pull request merged**